apps/omg_watcher/test/omg_watcher/exit_processor/core_test.exs

```diff
@@ -1325,22 +1325,126 @@ defmodule OMG.Watcher.ExitProcessor.CoreTest do
     end
   end
 
+  describe "determining utxos that are exited by finalization" do
+    test "returns utxos that should be spent when exit finalizes", %{
+      processor_empty: processor,
+      transactions: [tx1 | [tx2 | _]]
+    } do
+      ife_id1 = 1
+      ife_id2 = 2
+      tx_hash1 = Transaction.raw_txhash(tx1)
+      tx_hash2 = Transaction.raw_txhash(tx2)
+
+      processor =
+        processor
+        |> start_ife_from(tx1, status: {1, ife_id1})
+        |> start_ife_from(tx2, status: {1, ife_id2})
+
+      {processor, _} = Core.new_piggybacks(processor, [%{tx_hash: tx_hash1, output_index: 0}])
```

Comment:
with the increased amounts of calls to `Core.new_piggybacks`, I think it's time to do a refactoring due:

Follow the same approach as `start_ife_from(tx, status: {1, ife_id})` -> `piggyback_ife_from(tx, output_index)` so that we can append the `|> piggyback_ife_from(...)` to the pipes like above.

Direct calls to `Core.new_piggybacks` would stay on the few occasions where it is more convenient/ilustrative or where we want to do sth with the db updates

Reply:
Done. I left calls that use multiple piggybacks in argument list and those that test Core.new_piggybacks/2